tests/datatypes/test_bytes.py

```diff
@@ -28,6 +30,12 @@ def test_capitalize(self):
             print(b'2015638687'.capitalize())
         """)
 
+    @unittest.skip("Move this to test_capitalize upon resolution of #530")
```

Comment:
Don't be sorry, it was educational for me to hear this discussion on skip/skipIf v. expectedFailure! I've pushed the change. Hopefully that works. :)